### ISSUE TITLE
Windows 11 家庭中文版编译成功但是应用打开后没有vue部分的内容，只有main.go里设置的效果

### ISSUE THREAD

**Issue by yonomo** [wailsapp/wails#2781]
### Description

1、用的Windows 11 家庭中文版，但是wails doctor 检测为Windows 10 Home China
2、build成功后，exe文件打开后没有预期的内容

### To Reproduce

PS D:\test\wails\test2\helloworld> wails build 
DEB | Using go webview2loader
Wails CLI v2.5.1

# Build Options

Platform(s)        | windows/amd64
Compiler           | D:\devsoft\go\bin\go.exe
Skip Bindings      | false
Build Mode         | production
Frontend Directory | D:\test\wails\test2\helloworld\frontend
Obfuscated         | false
Skip Frontend      | false
Compress           | false
Package            | true
Clean Bin Dir      | false
LDFlags            |
Tags               | []
Race Detector      | false

# Building target: windows/amd64

  • Generating bindings: 
Done.
  • Installing frontend dependencies:
Done.
  • Compiling frontend:
Done.
  • Generating application assets:
Done.
  • Compiling application:
Done.
 INFO  Wails is now using the new Go WebView2Loader. If you encounter any issues with it, please report them to https://github.com/wailsapp/wails/issues/2004. You could also use the old legacy loader with `-tags native_webview2loader`, but keep in mind this will be deprecated in the near future.
Built 'D:\test\wails\test2\helloworld\build\bin\helloworld.exe' in 9.621s.

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony

### Expected behaviour

/

### System Details

```shell
PS D:\test\wails\test2\helloworld> wails doctor
DEB | Using go webview2loader
Wails CLI v2.5.1

 SUCCESS  Done.

# System

OS           | Windows 10 Home China
Version      | 2009 (Build: 22621)  
ID           | 22H2
Go Version   | go1.20.5
Platform     | windows
Architecture | amd64

# Wails

Version | v2.5.1

# Dependencies

Dependency | Package Name | Status    | Version      
WebView2   | N/A          | Installed | 114.0.1823.82
Nodejs     | N/A          | Installed | 20.4.0       
npm        | N/A          | Installed | 9.7.2        
*upx       | N/A          | Installed | upx 4.0.2    
*nsis      | N/A          | Installed | v3.09        
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```

**Comment by leaanthony** [comment]
The Windows 10 string is what Windows reports as the OS. Sounds crazy, but it's true.

**Comment by XudongLiu** [comment]
YES. I have the same problem, and dev is alway OK, but after building, the app can not run @yonomo Have you solved the problem?

**Comment by leaanthony** [comment]
Hey there\!

First off, thank you for taking the time to open this issue. Community feedback like yours has been instrumental in shaping Wails into what it is today.

We're doing some housekeeping on our issue tracker as we focus on the v3 release. This issue has been open for a while, and honestly, with the project's growth and the sheer volume of feedback we receive, some things inevitably slip through the cracks. That's on us, not you.

We're closing this issue, but that doesn't mean it wasn't valuable. If this is still relevant:

- For bugs: Please test with the latest version (or v3 alpha) and open a fresh issue if it persists
- For features: Feel free to open a new issue with updated context

Thank you for being part of the Wails journey. Your patience and support mean more than you know.
